### ISSUE TITLE
USB Type B power input on next PCB revision

### ISSUE THREAD

**Issue by vguttmann** [edson-acordi/4bit-microcomputer#5]
I think that a USB Type B connector like they use on the Arduino would be pretty convenient to have as a power input, since then a normal USB port (the USB specification specifies 1 Ampere max current if we exclude USB C and Power Delivery) should be able to power the computer just fine, and anyone that builds this likely also has an Arduino they connect with that cable.

**Comment by edson-acordi** [comment]
Yes, but you will need two cables, one to power the Arduino and the other to power the MikroLeo.
